docs/docs/reference/extractors.md

```diff
@@ -55,6 +55,54 @@ set the `record_format` to be `json` in the `StreamExtractor` configuration. For
      record_format: json
 ```
 
+## `QueueExtractor`
```

Comment:
Same thing for here: 

https://github.com/nodestream-proj/docs/blob/main/docs/reference/extractors.md

Reply:
I'll create a pr for the public docs after we finalize this pr.